storm-core/src/jvm/org/apache/storm/container/ResourceIsolationInterface.java

```diff
@@ -56,4 +64,13 @@
      */
     List<String> getLaunchCommandPrefix(String workerId);
 
+    /**
+     * Get the list of PIDs currently in an isolated container
+     * @param workerId the id of the worker to get these for
+     * @return the set of PIDs, this will be combined with
+     * other ways of getting PIDs. An Empty set or null if
+     * no PIDs are found.
+     * @throws IOException on any error
+     */
```

Comment:
There is only one caller currently, and I thought because this is pluggable and the caller is not, that being more lenient here would be better, but I am happy to change it if you feel strongly about it.